tests/pytest_tests/system_tests/test_artifacts/test_wandb_artifacts.py

```diff
@@ -1372,7 +1372,7 @@ def test_http_storage_handler_uses_etag_for_digest(
 def test_s3_storage_handler_load_path_missing_reference(monkeypatch, wandb_init):
     # Create an artifact that references a non-existent S3 object.
     artifact = wandb.Artifact(type="dataset", name="my-arty")
-    mock_boto(artifact)
+    mock_boto(artifact, version_id="")
```

Comment:
setting version_id="" here allows us to ignore the ObjectVersion path and lets the patched method kick in.